build.gradle.kts

```diff
@@ -21,9 +21,11 @@ dependencies {
 
    api("us.ihmc:ihmc-commons:0.32.0")
    api("us.ihmc:euclid-frame:0.21.0")
+//   api("org.ejml:ejml-ddense:0.39")
```

Comment:
Ideally we could verify whether or not this is actually required before merge.

Reply:
I did verify it is not needed and need to delete it, thank you